lib/nfc/nfc_worker.c

```diff
@@ -454,10 +452,7 @@ void nfc_worker_read_type(NfcWorker* nfc_worker) {
                     break;
                 }
             } else {
-                if(!card_not_detected_notified) {
-                    nfc_worker->callback(NfcWorkerEventNoCardDetected, nfc_worker->context);
-                    card_not_detected_notified = true;
-                }
+                nfc_worker->callback(NfcWorkerEventNoCardDetected, nfc_worker->context);
             }
```

Comment:
@gsurkov why do we need this change?
The idea was to send WorkerEventNoCardDetected event only once after card was lost. In this code we will always send WorkerEventNoCardDetected event if no card is detected

Reply:
The `card_not_detected_notified` variable is reset on every iteration and therefore does nothing. Perhaps there initially was a logical error.

Reply:
Corrected the logic, thanks